docs/cli/kops_create_instancegroup.md

```diff
@@ -36,7 +36,7 @@ kops create instancegroup [flags]
       --edit             If true, an editor will be opened to edit default values. (default true)
   -h, --help             help for instancegroup
   -o, --output string    Output format. One of json|yaml
-      --role string      Type of instance group to create (Node,Master,Bastion) (default "Node")
+      --role string      Type of instance group to create (Master,APIServer,Node,Bastion) (default "Node")
```

Comment:
Why do we write "APIServer" for the role name for _kops create instancegroup_, but write "Apiserver" for role name for _kops rolling-update cluster_?